src/components/SocialMediaFooterIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+export default function SocialMediaFooterIcon({
+  icon,
+}: {
+  icon: IconDefinition;
+}) {
+  const [changeColourOnHover, SetchangeColourOnHover] = useState(false);
```

Comment:
this isn't very clear, I would use something like isHovered

Reply:
addressed.

Reply:
from the other PR, I mentioned using is____ naming convention

Reply:
addressed correctly.